tests/functional/tests/file_manager/suite_file_manager.cpp

```diff
@@ -0,0 +1,78 @@
+// Leka - LekaOS
+// Copyright 2022 APF France handicap
+// SPDX-License-Identifier: Apache-2.0
+
+#include <cstddef>
+
+#include "FATFileSystem.h"
+#include "FileManagerKit.h"
+#include "SDBlockDevice.h"
+#include "boost/ut.hpp"
+#include "filesystem"
+#include "tests/config.h"
+#include "tests/ft_utils.h"
+
+using namespace leka;
+using namespace boost::ut;
+
+SDBlockDevice sd_blockdevice(SD_SPI_MOSI, SD_SPI_MISO, SD_SPI_SCK);
+FATFileSystem fatfs("fs");
+
+void initializeSD()
+{
+	constexpr auto default_sd_blockdevice_frequency = uint64_t {25'000'000};
+
+	sd_blockdevice.init();
+	sd_blockdevice.frequency(default_sd_blockdevice_frequency);
+
+	fatfs.mount(&sd_blockdevice);
+}
```

Comment:
I think this should be moved to `utils.h` with the same structure as

https://github.com/leka/LekaOS/blob/ad37a1c39559572db0bf0f2603959bb99d2a9d2f/app/os/main.cpp#L92-L110